index.html

```diff
@@ -0,0 +1,94 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+
+    <div class="main-page">
+
+      <div class="title">
+        <h1>Welcome to Microverse Remote Courses</h1>
+        <p class="description"> Thank you for taking the time to help us improve the platform</p>
+      </div>
+
```

Comment:
You can put this part within the <header> section as the requirement in Html

Reply:
- [ ] Using semantic tags is a best practice and it is great that you pointed this out for the developer. However, I would suggest that your feedback be more polite rather than an order. Also, it would be better to give a reason as to why the developer should use semantic tags.